### DISCUSSION TITLE
Ranked Choice Voting in Elections

### DISCUSSION THREAD

**Discussion by cmahns** [OpenSlides/OpenSlides#6628] [Ideas]
Short summary
-----
It would be really great if OpenSlides supported ranked choice voting methods for its "Elections" feature. Currently today it only supports simple tabulation methods which makes it not feasible for board elections or other officer positions for some political organizations. 

Context
-----
My organization has been using OpaVote for several years for sending ballots for board elections for local chapters, and national level leadership positions. It works well, but the cost involved is increasingly becoming prohibitive as the organization grows in scale. While many locals can use the Elections functionality as is for uncontested positions or for approval elections, we're bound by our bylaws to provide a [Hare based system](https://en.wikipedia.org/wiki/Hare_quota) if a minority of members demand for it prior to ballots being sent out ([Article VI Section 3 of our bylaws](https://www.dsausa.org/about-us/constitution/) if you want to read the full context), which means OpenSlides in its current state would no longer be able to be used and OpaVote would need to be substituted in at the last minute. From a cost perspective and member frustration perspective, this isn't great and it will likely lead to unintentional disenfranchisement. 

Below are two ranked choice methods that have historically been used, and one that is actively being evaluated by my organization: 
* [Borda Count](https://blog.opavote.com/2017/03/why-we-love-borda-count.html) (technically not a hare system but has a lot of proponents and is frequently used)
* [Scottish Single Transferable Vote (STV)](https://www.opavote.com/methods/scottish-stv-rules)
* [Proportional Score Then Automatic Runoff (STAR-PR)](https://www.starvoting.org/star-pr) (also not a hare system, but promising just for its simplicity)

We've had a really positive reception to OpenSlides by our members and this is the sole feature we're missing from being able to provide this as a complete suite for meetings and our bi-annual convention. Thank you again OpenSlides team for all the hard work that has gone in to this excellent software!

**Comment by jsangmeister** [comment]
Progress tracked in https://github.com/OpenSlides/OpenSlides/issues/6724
